_data/tools.yml

```diff
@@ -1338,7 +1338,7 @@
       url: https://wix-incubator.github.io/quix/docs/presto
 - name: R
   anchor: r
-  category: client
+  category: driver_unofficial
```

Comment:
Lets not overload category .. instead make a separate other field .. 

maybe 

owner: trinodb
owner: other

or 

trinodb-project: true|false